agent/exploits/cve_2019_12989.py

```diff
@@ -74,7 +77,7 @@ def check(self, target: definitions.Target) -> list[definitions.Vulnerability]:
                 return [vulnerability]
             else:
                 return []
-        except requests.exceptions.JSONDecodeError:
+        except json.JSONDecodeError:
```

Comment:
Do both request.exception main error and json decode error to support future requests versions.

Reply:
Also add unit test.